cfdi.go

```diff
@@ -284,6 +282,16 @@ func lookupTipoDeComprobante(inv *bill.Invoice) string {
 	return code.String()
 }
 
+func tipoCambio(inv *bill.Invoice) string {
+	for _, r := range inv.ExchangeRates {
```

Comment:
This method might be a bit nice to use: https://github.com/invopop/gobl/blob/main/currency/exchange_rate.go#L62

Reply:
Nice one! Addressed on b3427b3